legal-api/report-templates/template-parts/common/legalNameChange.html

```diff
@@ -0,0 +1,22 @@
+
+{% if alteration.nameRequest and alteration.nameRequest.legalName!=business.legalName %}
```

Comment:
either use the legalNameChange.html or you will need to open a techdebt ticket with Linda to alter the CoN filing output.

Reply:
I will use legalNameChange.html